Writing/Twelve principles of time management.md

```diff
@@ -0,0 +1,38 @@
+
+---
+tags: management, writing, time
+author: Tran Hoang Nam
+---
+
+**12 Principles Of Time Management**
+
+ - Determine what is urgent
+ - Don’t over commit
+ - Have a plan for your time
+ - Allow time for the unexpected
+ - Handle things once
+ - Create realistic deadlines
+ - Set goals for yourself and your time
+ - Develop routines
+ - Focus on one thing at a time
+ - Eliminate or minimize distractions
+ - Outsource tasks or delegate
+ - Leave time for fun and play
+---
+
+#### Pomodoro Technique
+The Pomodoro Technique is a time-management philosophy that aims to give the user maximum focus and creative freshness, enabling them to finish projects more quickly and with less mental exhaustion.
+
+It is an easy process. You allocate time for each project throughout the day in little bursts and take breaks as needed. 25 minutes of labor are followed by a 5-minute rest.
+
+A "pomodoro" is a 25-minute work block, so named after the Italian word for tomato. The method's name comes from Francesco Cirillo, who utilized a kitchen timer in the form of a tomato as his personal timer. You take a 15-20 minute break after four "pomodoros" (100 minutes of work time and 15 minutes of break time) have been completed.
+
+You mark your progress with a "X" after each completed pomodoro and keep track of how many times throughout each 25-minute period you were tempted to delay or switch to working on anything else.
```

Comment:
This share is ok but I think it lacks the explain why this technique is effective.